README.md

```diff
@@ -1 +1,104 @@
-# angular-ui-router-readme
\ No newline at end of file
+# What is ui-router?
+
+## Overview
+
+`ngRoute` provides us with basic functionality, but the leading router for Angular is `uiRouter`. This provides much more, such as nested views, multiple views and a few different offerings. Let's take a look at how `uiRouter` does things instead.
+
+## Objectives
+
+- Describe ui-router and ui-view
+- Write a route using ui-router
+- Integrate a Controller with ui-router
+- Integrate a template with ui-router
+
+## uiRouter
+
+`uiRouter` is quite similar to `ngRoute`, but each "route" is known as a "state". We define states, not routes.
+
+Each state that we define is quite similar to a route in `ngRoute`, but we have to give each state a name too. This allows us to have children states inside of a state.
```

Comment:
The last part of this line is a bit confusing - what do you mean by having children states inside of a state? Maybe mention that that's how we'll refer to this state in other parts of the application?